terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -382,8 +379,8 @@ locals {
   }
 
   umccrise_wfl_version = {
-    dev = "2.1.1--3.9.3"
-    prod = "2.1.1--3.9.3--e4195a6"
+    dev = "2.2.0--3.9.3"
+    prod = "2.2.0--3.9.3--4e00721"
```

Comment:
This branch is worked on top of 257. So, this is ok. Just noting.

Reply:
Have merged master back into this branch, so changes are now only relevant to this branch